src/counterexamples/canonically_ordered_comm_semiring_two_mul.lean

```diff
@@ -0,0 +1,281 @@
+/-
+Copyright (c) 2021 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import data.zmod.basic
+import ring_theory.subsemiring
+import algebra.ordered_monoid
+/-!
+
+A `canonically_ordered_comm_semiring` with two different elements `a` and `b` such that
+`a ≠ b` and `2 * a = 2 * b`.
+
```

Comment:
Could you explain the example in the file-level docstring? (And also recall the definition of `canonically_ordered_comm_semiring`)?

Reply:
Sébastien, I added a more detailed doc-string: was this what you had in mind?